### PR TITLE
Update documentation and remove legacy call

### PR DESCRIPTION
**Pull request by magarnier**
This commit:
 - remove or replace deprecated screenshot
 - update old stuff
 - add sections for portfolio and labels
 - reorder project admin links to be consistent with UI

### CODE REVIEW

File: languages/en/user-guide/project-labels.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.. _project-labels:
+
+
+Project Labels
+==============
+
+Projects labels enables you to sort items by labels.
```
Comment (nterray) [inline languages/en/user-guide/project-labels.rst]: "sort items by labels". Are you sure? How?

File: languages/en/user-guide/project-labels.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+.. NOTE::
+
+    For now labels are only available for pullrequest service
+
+Add a label
+-----------
+
+You can add labels on items by click on the "+" icon.
+If label does not exists in project, it will be automatically created.
+The input will propose you existing project labels.
```
Comment (nterray) [inline languages/en/user-guide/project-labels.rst]: s/will propose you/will suggest/

File: languages/en/user-guide/tracker.rst [hunk 3]
```diff
@@ -5,7 +5,7 @@ Trackers and real-time Reports
 
 **Disclaimer:** this chapter covers the new generation (aka v5) of
 Tracker system. For documentation on the legacy tracker system (v3) see
-?.
+user-guide/tracker-v3.
```
Comment (nterray) [inline languages/en/user-guide/tracker.rst]: It does not look like a rst reference.

File: languages/en/user-guide/citizen.rst [hunk 4]
```diff
@@ -96,55 +96,41 @@ To log into the Tuleap site, go to the Tuleap
...
 
-Tuleap allows you to customize your page to fit your
-needs. The content of your page is made of *widgets*. All widgets can be
-moved, collapsed or closed. Some of them can provide an RSS feed or
-preferences can be set.
+A sample user dashboard is available below. The page is split in several areas:
 
-New widgets can be added through the "Add widgets" link. Here is an
-incomplete list of available widgets for the personal page:
+-  The header with your user name and avatan
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/citizen.rst]: Done

File: languages/en/user-guide/citizen.rst [hunk 5]
```diff
@@ -96,55 +96,41 @@ To log into the Tuleap site, go to the Tuleap
...
+-  A navigation bar to access to each dashboard. Tthis is where you can add a new dashboard, and edit or delete a dashboard.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/citizen.rst]: Done

File: languages/en/user-guide/citizen.rst [hunk 6]
```diff
@@ -96,55 +96,41 @@ To log into the Tuleap site, go to the Tuleap
...
-   search in your project Bug or Task database, etc ... Bookmarked items
-   can be edited which means that both the title of the bookmark and its
-   destination URL can be modified.
-
 -  **My Projects** lists the projects you belong to. Selecting any of
    these projects brings you to the corresponding Project Dashboard.
 
+-  **Monitored File Packages** lists packages that you are currently
+   monitoring, by project. To cancel any of the monitored items just
+   click on the trash icon (|trash_icon|) next to the item label.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/citizen.rst]: Done

File: languages/en/user-guide/citizen.rst [hunk 7]
```diff
@@ -158,8 +144,7 @@ Account Maintenance
 You provided a certain amount of demographic information to
 Tuleap when you first registered on the Tuleap
 site. This and other information can be modified at any time by
-selecting the "My Account" item in the menu under your name (at the top right of
-the platform)
+click on the "cog" image near your user name.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/citizen.rst]: Done

File: languages/en/user-guide/frs.rst [hunk 8]
```diff
@@ -115,20 +115,13 @@ link.
...
-The first example given on `Example of a Delivery Manager screen`_ shows the Delivery Manager screen of the
-Playground project.
-
 Browsing packages
 `````````````````
 
-As explained in the `Delivery Manager Jargon`_, projects can contain several packages. In the
-example, the Playground project has four packages.
+As explained in the `Delivery Manager Jargon`_, projects can contain several packages.
+In order to have a better visibility on service, all packages are collapsed by default.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/frs.rst]: Done

File: languages/en/user-guide/frs.rst [hunk 9]
```diff
@@ -115,20 +115,13 @@ link.
...
 
-The content of each package can be hidden by clicking the *minus* icon
-located just before the package name. This can help for visibility
-reasons if your project has lots of packages and releases. To expand a
-collapsed package, simply click the *plus* icon before the package name.
-By default, all the packages are expanded.
+You can the full content of a package bu clicking on the $plus" icon located just before the package name.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/frs.rst]: Done

File: languages/en/user-guide/new-project.rst [hunk 10]
```diff
@@ -82,11 +75,10 @@ The project registration is an easy process:
 
    -  the documents, as well as the document tree will be present.
 
-   -  the CVS admin settings are copied (preamble, tracking mode, CVS
-      Watch Mode), but the emails and the CVS permissions are not
-      copied.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/new-project.rst]: Done

File: languages/en/user-guide/new-project.rst [hunk 11]
```diff
@@ -82,11 +75,10 @@ The project registration is an easy process:
...
+   -  the SVN admin settings are copied but
+      the emails and the SVN access permissions are not copied.
 
-   -  the SVN admin settings are copied (preamble, tracking mode), but
+   -  the Git admin settings and global permissions are copied, but
       the emails and the SVN access permissions are not copied.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/new-project.rst]: Done

File: languages/en/user-guide/new-project.rst [hunk 12]
```diff
@@ -103,22 +95,12 @@ The project registration is an easy process:
...
-   Please select up to three classifications for this project in each of
-   the Trove root categories. This will help potential developers and
-   users to find your project and engage. If the project does not
-   require any or all of these classification, simply select "None
-   Selected". If you used a project template that had already been
-   classified you will find its categories already selected here. You
-   can change the trove categories for this project at any time. You can
-   categorize your project according to the seven criteria listed in :ref:`software-map-(or Project Tree)`.
+   Some categories can be mandatory for project creation.
+   Theses categories are managed by your system administrator.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/new-project.rst]: Done

File: languages/en/user-guide/new-project.rst [hunk 13]
```diff
@@ -182,7 +164,7 @@ after a project is registered:
     Tuleap we'll refine the hierarchy. If you cannot find
     the appropriate topic for your project, please contact us through
     the "Contact Us" link in the Tuleap Main Menu or submit
-    a request in the appropriate Discussion Forum.
+    a request in the appropriate our community chat.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/new-project.rst]: Done

File: languages/en/user-guide/overview.rst [hunk 14]
```diff
@@ -35,160 +32,39 @@ in the document in the related service description.
...
-The Software Map is a pivotal service in Tuleap that you can
-(should) use to determine if there are some pieces of software that you
-can re-use for your own work. The Software Map is one of the 2 ways by
-which you can search for existing software projects. The other one
-relies on the Search Facility.
+Software map will allow you to categorize your project in categories.
+For instance Tuleap comes with a Licence > Open source Approved License category.
+
+As a project administrator you are able to sort all your projects
+group by open source or not licenced.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/overview.rst]: Done

File: languages/en/user-guide/overview.rst [hunk 15]
```diff
@@ -35,160 +32,39 @@ in the document in the related service description.
...
-were created primarily as a communication vehicle between the
-Tuleap users and the Tuleap Team. These are
-general purpose Forums where you can ask for Help with
-Tuleap, suggest new features, new categories for the
-software map, etc.
+API
+~~~
+You will find here documentation for our APIS:
+  * REST API
+  * SOAP API (deprecated)
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/overview.rst]: Done

File: languages/en/user-guide/portfolio.rst [hunk 16]
```diff
@@ -0,0 +1,19 @@
+Portfolio
+=========
+
+You may want perform a search based on several trackers and several projects.
+A widget `Cross tracker search` enables you to do that.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/portfolio.rst]: Done

File: languages/en/user-guide/portfolio.rst [hunk 17]
```diff
@@ -0,0 +1,19 @@
...
+
+.. NOTE::
+  This widget is still under development, for now it only allow you to search
+  only the 30 first `Open` artifacts corresponding to the query.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/portfolio.rst]: Done

[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/portfolio.rst]: Done

File: languages/en/user-guide/project-admin.rst [hunk 18]
```diff
@@ -10,6 +10,39 @@ menu.
...
+-  **Test Project**: If you just need to test what is feasible with
+   Tuleap projects use the Test Project type for your
+   project. Having defined that type, your project will not appear any
+   more in the Tuleap Software Map.
+
+
+Project Visibility
+``````````````````
+
+You can change your project visibility at any time.
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/project-admin.rst]: Done

File: languages/en/user-guide/project-labels.rst (hunk 1 above)
Comment (magarnier) [reply languages/en/user-guide/project-labels.rst]: Done

File: languages/en/user-guide/project-labels.rst (hunk 2 above)
Comment (magarnier) [reply languages/en/user-guide/project-labels.rst]: Done

File: languages/en/user-guide/overview.rst [hunk 19]
```diff
@@ -35,160 +32,34 @@ in the document in the related service description.
...
-Administration project, which is of course hosted on Tuleap.
-The documents on this page cover all the Tuleap site. It
-includes presentation material, the user guide and other important
-documents. Read them carefully.
 
-Tuleap Developers Channels
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+API
+~~~
+You will find here documentation for our REST API:
```
Comment (nterray) [inline languages/en/user-guide/overview.rst]: Trailing colon to be removed

File: languages/en/user-guide/tracker.rst [hunk 20]
```diff
@@ -704,37 +704,8 @@ Linked or "to be linked" artifacts can be managed in the artifact link
 tabs view. Not yet linked artifacts are highlighted. All artifacts can
 be unckeck (by clicking the red cross).
 
-Using typed artifact links
```
Comment (nterray) [inline languages/en/user-guide/tracker.rst]: Is it possible to revert the modifications done on tracker.rst from this PR?

File: languages/en/user-guide/project-admin.rst [hunk 21]
```diff
@@ -180,6 +362,21 @@ the database. The service data are however preserved.
 System services cannot be deleted. However, you can disable them and
 they will not appear in the sidebar.
 
+
+Label
+-----
+You can administrate here all your project labels by:
```
[the remark this answers is not in the document]
Comment (magarnier) [reply languages/en/user-guide/project-admin.rst]: Done

**Pull request merged**